### ISSUE TITLE
New version of Sonos app

### ISSUE THREAD

**Issue by salteggs** [dkanada/frost#371]
In June this year, Sonos released a new version of their app, called Sonos S2, and renamed their original app to Sonos S1.

Some of the most fundamental changes, in the [FAQ on their website](https://www.sonos.com/en-us/controller-app):
- Is Sonos ending support for older products?
* No. You can still use older products with the S1 Controller app. The S1 Controller app will receive bug fixes and security patches, but it will not receive future software features.
- Are any products incompatible with the S1 Controller app?
* Products released after May 2020 will not be compatible with the S1 Controller app. This includes the new Arc, Five, and Sub (Gen 3). Product compatibility will be noted on sonos.com.

Basically, if you have both new Sonos products and legacy Sonos products, you need to use 2 separate apps to control them all.

Since Frost has a Sonos logo for the S1 app `com.sonos.acr`, the S2 app `com.sonos.acr2` is the default dark orange logo. My first thought was to copy the existing logo for the S2 app but that might be confusing for the few people using both apps, if any.

What are your thoughts? Copy it over anyway? Or maybe adding S1 / S2 in the bottom of the logo?

Edit, turtl info: `<item component="ComponentInfo{com.sonos.acr2/com.sonos.acr.SonosLaunchActivity}" drawable="sonos" />`

**Comment by Donnnno** [comment]
Hey, I think just copying the icon for the S2 app would be the easiest thing to. It also keeps the icon minimalistic (that has my preference)

**Comment by salteggs** [comment]
Sounds good to me.

The amount of people using both apps should be very slim. And given some time, they'll probably know which app to open, or have both open anyway.

**Comment by Donnnno** [comment]
Cool! I'll add the line to my pull request.

**Comment by salteggs** [comment]
Oh, did not notice you added the line already, cool! Thank you!
Good luck with the pull request.

**Comment by salteggs** [comment]
FYI, it works great :+1:

**Comment by Donnnno** [comment]
Cool! Good to know ;)
